src/Mvc/Mvc.ViewFeatures/src/Rendering/IPrerenderingDependencyManager.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Mvc.Rendering
+{
+    /// <summary>
+    /// Manages prerendering dependencies to ensure that components are prerendered in the
+    /// desired order.
+    /// </summary>
+    public interface IPrerenderingDependencyManager
```

Comment:
I'm open to name changes, since this name might suggest that more is going on than what's actually happening.

Reply:
Ok, I guess this is still WIP because I don't see where these APIs are being called from.